models/src/main/scala/coop/rchain/models/ByteArraySyntax.scala

```diff
@@ -12,4 +13,6 @@ trait ByteArraySyntax {
 class ByteArrayOps(private val ba: Array[Byte]) extends AnyVal {
   def toByteString: ByteString = ByteString.copyFrom(ba)
   def toHexString: String      = Base16.encode(ba)
+  def toParUnforgeableName: Par =
+    Par(unforgeables = Seq(GUnforgeable(GPrivateBody(GPrivate(ByteString.copyFrom(ba))))))
```

Comment:
RhoType can be used instead.
https://github.com/rchain/rchain/blob/0bfc2aa13e73a27c1f21200dee7336bd252ea17a/rholang/src/main/scala/coop/rchain/rholang/interpreter/RhoType.scala#L102-L110

You can also add overload for `Name`.
```scala
def apply(gprivateBytes: Array[Byte]): Par = apply(GPrivate(ByteString.copyFrom(gprivateBytes)))

// With that unforgeable name can be created simply without the need for extension
val unforgName = Name(<array_byte>)
```

Reply:
Dependency impossible to introduce rholang here.

Reply:
I'm not saying to change the extension method, but to remove it and use `Name(<array_byte>)` constructor instead.

Reply:
Please remove this unused extension method.